templates/S3/synapse-external-bucket.j2

```diff
@@ -151,25 +152,8 @@ Resources:
                 - "s3:PutObject"
                 - "s3:PutObjectAcl"
                 - "s3:DeleteObject"
+                - "s3:*MultipartUpload*"
```

Comment:
If you are combining the two statements, then perhaps rename InternalPutObjectAccess to simply bePutObjectAccess.

Reply:
fixed